src/test/java/com/intive/patronage22/szczecin/retroboard/service/BoardServiceTest.java

```diff
@@ -359,7 +356,8 @@ void patchBoardShouldUpdateNameAndNumberOfVotesFields() {
         final var email = "username@test.pl";
         final var boardName = "My first board.";
         final var userOwner = new User(uid, email, "displayName", Set.of());
-        final var board = buildBoard(userOwner);
+        final var state = EnumStateDto.CREATED;
```

Comment:
you don't need to create state as separate variable if it's used only in line below
```suggestion
        final var board = buildBoard(user, EnumStateDto.CREATED);
```

Reply:
done